scripts/styleChecker.fsx

```diff
@@ -192,7 +197,14 @@ let StyleXamlFiles() =
     Process
         .Execute(
             {
-                Command = "./node_modules/.bin/prettier"
+                Command =
+                    Path.Combine
+                        [|
+                            "."
```

Comment:
@realmarv wrong, you have to use `Directory.GetCurrentDirectory()` instead of `.`

Reply:
Should I return the used command in StyleXFiles so we can use the command in the suggestion string of CheckStyleOfXFiles?

Reply:
no because in the suggestion we want to use `.`

Reply:
Done